Makefile

```diff
@@ -306,6 +306,17 @@ install: install-sdk
 test:
 	(cd $(BUILD)/$(HOST) && ninja check_slow check_fuzzer_lib)
 
+.PHONY: test-serial
+test-serial:
+	@if [ -z "$$HW_TOIT_EXE" ]; then \
+		echo "HW_TOIT_EXE is not set. Set it to the path of the PI executable."; \
```

Comment:
```suggestion
		echo "HW_TOIT_EXE is not set. Set it to the path of the Pi executable."; \
```

Reply:
removed the second part of the sentence.
done.